docs/BUILD.md

```diff
@@ -0,0 +1,41 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Release workflow](#release-workflow)
+  - [Step 1: Release](#step-1-release)
+  - [Step 2: Publish](#step-2-publish)
+  - [Checklist](#checklist)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Release workflow
+
+The main goal of the build process is to build and publish the docker image to the docker hub repo: 'commercetools/commercetools-project-sync'
```

Comment:
Same as you are doing in the checklist, maybe link the repo URL ?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/231/commits/62dbc8c5442537f20d5d660b87ad830838dd5ea1